src/main/java/com/adobe/cq/commerce/graphql/client/GraphqlClient.java

```diff
@@ -112,4 +112,14 @@ public interface GraphqlClient {
      */
     public <T, U> GraphqlResponse<T, U> execute(GraphqlRequest request, Type typeOfT, Type typeOfU, RequestOptions options);
 
+    /**
+     * Delete cache entries for in-memory cache based on the cache entries.
```

Comment:
This is a bit confusing. Could you reformulate it a b it more clearly what this method is supposed to do?
The idea was that when all params are null, it clears everything, when some params are not null it clears the machine entries, right?

Reply:
Done